pkgs/top-level/all-packages.nix

```diff
@@ -22650,6 +22650,7 @@ in
     timepp = callPackage ../desktops/gnome-3/extensions/timepp { };
     topicons-plus = callPackage ../desktops/gnome-3/extensions/topicons-plus { };
     window-corner-preview = callPackage ../desktops/gnome-3/extensions/window-corner-preview { };
+    wsmatrix = callPackage ../desktops/gnome-3/extensions/wsmatrix.nix { };
```

Comment:
Can you move `wsmatrix.nix` to `wsmatrix/default.nix` to match the convention of the other extensions?